no VARIANT on amd64

```
:; GOARCH=arm64 go build
# github.com/go-ole/go-ole
./com_func.go:105:21: undefined: VARIANT
./com_func.go:110:22: undefined: VARIANT
./connect.go:79:72: undefined: VARIANT
./connect.go:90:76: undefined: VARIANT
./connect.go:105:69: undefined: VARIANT
./connect.go:115:73: undefined: VARIANT
./connect.go:129:69: undefined: VARIANT
./connect.go:139:73: undefined: VARIANT
./connect.go:173:84: undefined: VARIANT
./idispatch.go:26:90: undefined: VARIANT
./idispatch.go:26:90: too many errors
```

--- a/variant_arm64.go
+++ b/variant_arm64.go
@@ -1,0 +1,13 @@
+//go:build arm64
+// +build arm64
+
+package ole
+
+type VARIANT struct {
+	VT         VT      //  2
+	wReserved1 uint16  //  4
+	wReserved2 uint16  //  6
+	wReserved3 uint16  //  8
+	Val        int64   // 16
+	_          [8]byte // 24
+}

--- a/variant_date_arm64.go
+++ b/variant_date_arm64.go
@@ -1,0 +1,23 @@
+//go:build windows && arm64
+// +build windows,arm64
+
+package ole
+
+import (
+	"errors"
+	"syscall"
+	"time"
+	"unsafe"
+)
+
+// GetVariantDate converts COM Variant Time value to Go time.Time.
+func GetVariantDate(value uint64) (time.Time, error) {
+	var st syscall.Systemtime
+	v1 := uint32(value)
+	v2 := uint32(value >> 32)
+	r, _, _ := procVariantTimeToSystemTime.Call(uintptr(v1), uintptr(v2), uintptr(unsafe.Pointer(&st)))
+	if r != 0 {
+		return time.Date(int(st.Year), time.Month(st.Month), int(st.Day), int(st.Hour), int(st.Minute), int(st.Second), int(st.Milliseconds/1000), time.UTC), nil
+	}
+	return time.Now(), errors.New("Could not convert to time, passing current time.")
+}

